### ISSUE TITLE
RETAIN: Interpretable Predictive Model in Healthcare using Reverse Time Attention Mechanism

### ISSUE THREAD

**Issue by blengerich** [greenelab/deep-review#189]
[https://arxiv.org/pdf/1608.05745v3.pdf](https://arxiv.org/pdf/1608.05745v3.pdf)

TL;DR: Invert typical RNN+attention mechanism stack to identify significant clinical variables in EHR data.

Would this paper be appropriate for either the EHR subsection of the "categorize" section or the Clinical Decision Making subsection of the "treat" section? It doesn't directly match patients with treatments, but it does prioritize clinical interpretability

**Comment by agitter** [comment]
We haven't fully scoped the clinical decision making subsection so I'm not sure what exactly will go there.  This is potentially a good fit.  Are you interested in helping write about this, or other, topics?

Tagging @AvantiShri and @akundaje for relevance to the interpretation Discussion section as well.

**Comment by blengerich** [comment]
Yes, I would be interested in helping to write about this. I can start a draft of the "Categorize" subsection of the treat section if you would like. If someone is already working on this, I'd also be willing to help write elsewhere.

**Comment by agitter** [comment]
That would be excellent, no one is working on that to my knowledge.  I'll update #188.
